backend/LexBoxApi/GraphQL/UserMutations.cs

```diff
@@ -105,6 +108,12 @@ EmailService emailService
             await emailService.SendVerifyAddressEmail(user, input.Email);
         }
 
+        if (wasPromotedToAdmin)
+        {
+            var oldAdmins = dbContext.Users.Where(u => u.IsAdmin).AsAsyncEnumerable();
```

Comment:
I get the impression that at one point you were considering only notifying the old-admins, but I think this now also contains the new admin.
```suggestion
            var admins = dbContext.Users.Where(u => u.IsAdmin).AsAsyncEnumerable();
```

Reply:
Done in commit c105ef86.